app/Services/ProductionRequestActivityLogger.php

```diff
@@ -0,0 +1,19 @@
+<?php 
+
+namespace App\Services;
+use Illuminate\Http\Request;
+
+class ProductionRequestActivityLogger extends AbstractRequestActivityLogger
+{
+    protected function collectRequestData(Request $request): array
+    {
+        return [
+            "name" => $request->input('name'),
```

Comment:
You should not log input fields in production logger, just important information about request.

Reply:
corrected